src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -358,7 +358,10 @@ export class GwtCallback extends EventEmitter {
     });
 
     ipcMain.on('desktop_show_word_doc', (event, wordDoc: string) => {
-      GwtCallback.unimpl('desktop_show_word_doc');
+      shell.openPath(resolveAliasedPath(normalizeSeparatorsNative(wordDoc))).catch((value) => {
+        console.log('error:', value);
+        logger().logErrorMessage(value);
```

Comment:
The Qt code falls back to showing the file in the View pane.

I think we can also add this change to the NEWS since it used to only open in the View pane on non-Windows platforms. I'm not sure why this wasn't done before since it should work on all platforms that have an application that can read docx.

Reply:
where are you seeing in the Qt code that it opened in the view pane on failure? I just see it calling the showFile method which just does 
```
void GwtCallback::showFile(QString path)
{
   if (path.isNull() || path.isEmpty())
      return;

   path = desktop::resolveAliasedPath(path);

   desktop::openUrl(QUrl::fromLocalFile(path));
}

void GwtCallback::showWordDoc(QString path)
{
#ifdef Q_OS_WIN32

   path = desktop::resolveAliasedPath(path);
   Error error = wordViewer_.showItem(path.toStdWString());
   if (error)
   {
      LOG_ERROR(error);
      showFile(path);
   }

#else
   // Invoke default viewer on other platforms
   showFile(path);
#endif
}
```
Not clear to me where it's falling back to opening in the viewer pane (can we even open a .docx file in the view pane?)

Reply:
I think you can use either the secondary or satellite window to do this. You might want to check how popping out an editor works to confirm this.

Reply:
sorry, that's really not clear to me -- I'm not following where this is done in the original Qt code, where there doesn't seem to be any kind of fallback for non-windows platforms, and there doesn't seem to be any way to open these kinds of files in the viewer pane currently. Opening the file through the File -> Open File menu or through the File pane will always open an external application. Maybe I'm missing what you're saying